gapic-generator-java/DEVELOPMENT.md

```diff
@@ -99,24 +99,12 @@ To generate a production GAPIC API:
 
    Replace the following section in googleapis
    ```
-    _gapic_generator_java_version = "2.13.0"
-
-    maven_install(
-        artifacts = [
-            "com.google.api:gapic-generator-java:" + _gapic_generator_java_version,
-        ],
-        #Update this False for local development
-        fail_on_missing_checksum = True,
-        repositories = [
-            "m2Local",
-            "https://repo.maven.apache.org/maven2/",
-        ]
-    )
+    _gapic_generator_java_version = "2.43.0"
     
     http_archive(
         name = "gapic_generator_java",
-        strip_prefix = "sdk-platform-java-%s" % _gapic_generator_java_version,
-        urls = ["https://github.com/googleapis/sdk-platform-java/archive/v%s.zip" % _gapic_generator_java_version],
+        strip_prefix = "sdk-platform-java-test-grpc",
```

Comment:
I might missed this in previous reviews, does it mean we are always loading gapic-generator-java jar from this branch? We are not loading the jar with `_gapic_generator_java_version` specified in previous lines?

Reply:
Ah, thanks for catching this. I am raising a cl to change it back.